releases/release-1.22/release-notes/maps/pr-90368-map.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 90368
+releasenote:
+  text: '''kube-controller-manger`: the `--horizontal-pod-autoscaler-use-rest-clients`
```

Comment:
We should also be careful how we start this note. and it should be in code block